ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/Main.scala

```diff
@@ -25,7 +26,6 @@ import scalaz.std.scalaFuture._
 import scalaz.std.string._
 import scalaz.syntax.tag._
 import scalaz.{-\/, EitherT, \/, \/-}
-
 import Config.QueryStoreIndex
```

Comment:
If possible, make sure that in the end the whitespace changes are minimal (If you do that already, sorry for calling you out here. I haven't reviewed your code before).

Reply:
cheers will look at cleaning this up